buildpacks/nodejs-corepack/tests/snapshots/corepack_npm_8.snap

```diff
@@ -7,7 +7,7 @@ source: test_support/src/lib.rs
   - Node.js version not specified, using `22.x`
   - Resolved Node.js version: `22.16.0`
 - Installing Node.js distribution
-  - GET https://nodejs.org/download/release/v22.16.0/node-v22.16.0-<arch>.tar.gz ... (<time_elapsed>)
+  - GET https://nodejs.org/download/release/v22.16.0/node-v22.16.0-<arch>.tar.gz ... (OK)
```

Comment:
To consider for later: There's a "url" style that wraps in backticks and colorized. The output is coming from a `Url` type so that would catch any stray extra spaces and error on them, so it's not critical.